### PR TITLE
fix(LinkChooser): add missing https to custom link

### PR DESCRIPTION
**Pull request by Walkeezy**
Resolves issue `Adding a custom URL leads the user to a 404 Page`
Ticket: https://app.clickup.com/t/2a1aa72

### CODE REVIEW

File: src/components/LinkChooser/utils/helpers.ts [hunk 1]
```diff
@@ -30,3 +30,11 @@ export const isCustomLink = (link: SearchResult | null) =>
 
 export const filterItems = (query: string, results: SearchResult[], section: ExtraSection[] = []): SearchResult[] =>
     results.filter((item) => doesContainSubstring(item.title, query, section));
+
+export const prependHttps = (link: string) => {
+    if (link.indexOf("http://") === 0 || link.indexOf("https://") === 0) {
```
Comment (triggertoo) [inline src/components/LinkChooser/utils/helpers.ts:35]:
```suggestion
    if (!link.match(/https?:\/\//)) {
```

File: src/components/LinkChooser/utils/helpers.ts [hunk 2]
```diff
@@ -30,3 +30,11 @@ export const isCustomLink = (link: SearchResult | null) =>
...
+        return link;
+    } else {
```
Comment (triggertoo) [inline src/components/LinkChooser/utils/helpers.ts:37]: else can be removed

File: src/components/LinkChooser/utils/helpers.ts (hunk 1 above)
Comment (Walkeezy) [reply src/components/LinkChooser/utils/helpers.ts:35]: Nice, changed 👍🏻

File: src/components/LinkChooser/utils/helpers.ts (hunk 2 above)
Comment (Walkeezy) [reply src/components/LinkChooser/utils/helpers.ts:37]: Removed

**Review by NadiaPosch — APPROVED** [review]

**Pull request merged**